presto-docker/etc/node.properties

```diff
@@ -0,0 +1,5 @@
+node.environment=docker
+node.id=ffffffff-ffff-ffff-ffff-ffffffffffff
```

Comment:
Let's make this something like `dummy-node-id` so it's obvious that this should be changed

Reply:
Done.